src/renderer/store/reducers.ts

```diff
@@ -0,0 +1,47 @@
+import { combineReducers } from 'redux';
+import {
+  PROJECT_CREATED,
+  CURRENT_PROJECT_CLOSED,
+  RECENT_PROJECT_ADDED,
+  Action,
+} from './actions';
+import { ApplicationStore, initialStore, Project } from './helpers';
+
+const currentProjectReducer: (
+  currentProject: ApplicationStore['currentProject'],
+  action: Action<any>
+) => ApplicationStore['currentProject'] = (
+  currentProject = initialStore.currentProject,
+  action
+) => {
+  if (action.type === PROJECT_CREATED) {
```

Comment:
Just because this will be a template for future reducers, would it be a good idea to have a switch statement instead of multiple if statements - for readability ?

Reply:
To be honest I prefer if because switch kind of forces a particular format that gets kind of crowded if the logic gets complex

How about we leave it as if for now and if switch makes more sense we can change it later?

Reply:
Yep, I'm okay with that